.github/workflows/auto-publish.yml

```
@@ -98,6 +98,11 @@ jobs:
             echidna_token: ECHIDNA_TOKEN_HEVC_REGISTRATION
             build_override: |
               status: NOTE-WD
+          - source: video_frame_metadata_registry.src.html
+            destination: video_frame_metadata_registry.html
+            echidna_token: ECHIDNA_TOKEN_VIDEOFRAMEMETADATA_REGISTRY
+            build_override: |
+              status: NOTE-WD
```

If we're going to publish this document on the Registry Track, published status will be "Draft Registry"

```suggestion
              status: DRY
```

Note publication to /TR won't work until after the document has been manually published once to /TR. That's normal.

---

I reused the codec registry status in this file. I guess it should be changed for the codec registry then.

---

The codec registry is not yet on the Registry track (first publication was before the introduction of the Registry track in the Process document). The group resolved to move it to the Registry track, so status will indeed soon change.